pylint/plugins/hass_imports.py

```diff
@@ -398,6 +398,12 @@ class ObsoleteImportMatch:
             constant=re.compile(r"^save_json|find_paths_unserializable_data$"),
         ),
     ],
+    "typing_extensions": [
+        ObsoleteImportMatch(
+            reason="should be imported from collections.abc",
+            constant=re.compile(r"^(Async)?Generator$"),
+        ),
+    ],
```

Comment:
Reading up on https://docs.astral.sh/ruff/rules/deprecated-import/, I wonder if we should do this or not.
Maybe it is sufficient to just monitor it manually?

Reply:
```suggestion
```